internal/runbits/runtime/rationalize.go

```diff
@@ -47,9 +49,17 @@ func rationalizeError(auth *authentication.Auth, proj *project.Project, rerr *er
 				errNoMatchingPlatform.HostPlatform, errNoMatchingPlatform.HostArch,
 				proj.BranchName(), strings.Join(branches, "\n - ")))
 		} else {
-			*rerr = errs.NewUserFacing(locale.Tr(
-				"err_no_platform_data_remains",
-				errNoMatchingPlatform.HostPlatform, errNoMatchingPlatform.HostArch))
+			if errNoMatchingPlatform.LibcVersion != "" {
```

Comment:
Nathan created `errs.SetIf()` for this this kind of `if` statement: https://github.com/ActiveState/cli/blob/301fe307423cc054daf44f06eea26e95c3c75bf8/internal/runbits/checkout/rationalize.go#L26. Can we do something like:

```
libCError := errNoMatchingPlatform.LibcVersion != ""
*rerr = errs.NewUserFacing(
	locale.Tr("err_no_platform_data_remains", errNoMatchingPlatform.HostPlatform, errNoMatchingPlatform.HostArch),
	errs.SetIf(libCError, errs.SetInput()),
	errs.SetIf(libCError, errs.SetTips(locale.Tr("err_user_libc_solution", api.GetPlatformURL(fmt.Sprintf("%s/%s", proj.NamespaceString(), "customize")).String()))),
)
```
?

Reply:
Very nice! Didn't know about this and improves the readability quite a bit!